test/trackler/problem_test.rb

```diff
@@ -4,41 +4,109 @@
 class ProblemTest < Minitest::Test
   def test_existing_problem
     problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
-
     assert problem.exists?, "hello world files not found in the metadata dir"
+  end
+
+  def test_slug
+    problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
     assert_equal 'hello-world', problem.slug
+  end
+
+  def test_name
+    problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
     assert_equal 'Hello World', problem.name
+  end
+
+  def test_blurb
+    problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
     assert_equal 'Oh, hello.', problem.blurb
+  end
+
+  def test_description
+    problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
     assert_equal "* hello\n* hello again\n", problem.description
+  end
+
+  def test_metadata_url
+    problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
+    assert_equal 'https://github.com/exercism/x-common/blob/master/exercises/hello-world/metadata.yml', problem.metadata_url
+  end
+
+  def test_yaml_url_deprecated
+    problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
     assert_equal 'https://github.com/exercism/x-common/blob/master/exercises/hello-world/metadata.yml', problem.yaml_url
+  end
+
+  def test_description_url
+    problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
+    assert_equal 'https://github.com/exercism/x-common/blob/master/exercises/hello-world/description.md', problem.description_url
+  end
+
+  def test_md_url_deprecated
```

Comment:
It's the method, I'll clarify the test names, thanks for pointing out the ambiguity.